src/RackspaceAdapter.php

```diff
@@ -162,7 +162,23 @@ public function deleteDir($dirname)
      */
     public function createDir($dirname, Config $config)
     {
-        return ['path' => $dirname];
+        $headers = [];
+
+        if ($config->has('headers')) {
+            $headers = $config->get('headers');
+        }
+
+        if (!key_exists('Contents', $headers)) {
+            $headers['Contents'] = 'None';
```

Comment:
I wanted it to be dynamic, let's say the headers are already send through the config (for some reason). Maybe a implementation based on RackSpace but with slidely different values, one would be able to pass this through the config without being worried it's overwritten with 'None', the same goes for 'application/directory' in the Content-Type.